src/help-widget.spec.ts

```diff
@@ -510,4 +524,17 @@ describe('BBHelpHelpWidget', () => {
     expect(helpWidget['container'].classList).toContain('bb-help-container-mobile');
     done();
   });
+
+  it('should serialize json', () => {
```

Comment:
this doesn't really explain what it's doing.  Perhaps something like `should sanitize config options of functions`,